domain/src/test/java/org/oppia/android/domain/onboarding/DeprecationControllerTest.kt

```diff
@@ -138,6 +167,120 @@ class DeprecationControllerTest {
       )
   }
 
+  @Test
+  fun testController_ifUserNotOnboarded_userNotOnboardedStartUpMode() {
```

Comment:
The test names typically follow a format: `action_conditionOne_conditionTwo_expectedOutcome`
```suggestion
  fun testController_userNotOnboarded_returnsUserNotOnboardedStartUpMode() {
```

Reply:
Please review other test names as well.

Reply:
Done